internal/cmd/perfcomp/main.go

```diff
@@ -305,27 +318,31 @@ func getEnergyStatsForAllBenchMarks(ctx context.Context, patchRawData []RawData,
 
 func generatePRComment(energyStats []*EnergyStats, version string) string {
 	var comment strings.Builder
-	comment.WriteString("# 👋GoDriver Performance\n")
-	fmt.Fprintf(&comment, "The following benchmark tests for version %s had statistically significant changes (i.e., |z-score| > 1.96):\n", version)
+	fmt.Fprintf(&comment, "The following benchmark tests for version %s had statistically significant changes (i.e., |z-score| > 1.96):\n\n", version)
 
 	w := tabwriter.NewWriter(&comment, 0, 0, 1, ' ', 0)
 	fmt.Fprintln(w, "| Benchmark\t| Measurement\t| H-Score\t| Z-Score\t| % Change\t| Stable Reg\t| Patch Value\t|")
 	fmt.Fprintln(w, "| ---------\t| -----------\t| -------\t| -------\t| --------\t| ----------\t| -----------\t|")
 
-	var testCount int64
+	var significantEnergyStats []EnergyStats
 	for _, es := range energyStats {
-		if math.Abs(es.ZScore) > 1.96 {
-			testCount += 1
-			fmt.Fprintf(w, "| %s\t| %s\t| %.4f\t| %.4f\t| %.4f\t| Avg: %.4f, Med: %.4f, Stdev: %.4f\t| %.4f\t|\n", es.Benchmark, es.Measurement, es.HScore, es.ZScore, es.PercentChange, es.StableRegion.Mean, es.StableRegion.Median, es.StableRegion.Std, es.MeasurementVal)
+		if es.Measurement != "iterations" && math.Abs(es.ZScore) > 1.96 {
+			significantEnergyStats = append(significantEnergyStats, *es)
 		}
 	}
-	w.Flush()
 
-	if testCount == 0 {
+	if len(significantEnergyStats) == 0 {
 		comment.Reset()
-		comment.WriteString("# 👋GoDriver Performance\n")
-		comment.WriteString("There were no significant changes to the performance to report.")
+		fmt.Fprintf(&comment, "There were no significant changes to the performance to report for version %s.\n", version)
+	} else {
+		sort.Slice(significantEnergyStats, func(i, j int) bool {
+			return math.Abs(significantEnergyStats[i].PercentChange) > math.Abs(significantEnergyStats[j].PercentChange)
+		})
+		for _, es := range significantEnergyStats {
+			fmt.Fprintf(w, "| %s\t| %s\t| %.4f\t| %.4f\t| %.4f\t| Avg: %.4f, Med: %.4f, Stdev: %.4f\t| %.4f\t|\n", es.Benchmark, es.Measurement, es.HScore, es.ZScore, es.PercentChange, es.StableRegion.Mean, es.StableRegion.Median, es.StableRegion.Std, es.MeasurementVal)
```

Comment:
Optional: I'm not familiar with how to interpret the H and Z scores, so they're not currently useful when reviewing performance diffs. Consider moving them to the last two columns, after the PR measurement value.